infra/aws/terraform/prow-build-cluster/vpc.tf

```diff
@@ -38,10 +38,17 @@ module "vpc" {
   # (https://registry.terraform.io/modules/terraform-aws-modules/vpc/aws/latest#private-versus-intra-subnets)
   intra_subnets = var.vpc_intra_subnet
 
-  # Enable IPv6 for this subnet.
-  enable_ipv6                     = true
-  assign_ipv6_address_on_creation = true
-  create_egress_only_igw          = true
+  # Enable public IPv4 addresses
+  map_public_ip_on_launch = true
```

Comment:
is this necessary? its generally less secure and they will start to incur charges soon https://aws.amazon.com/blogs/aws/new-aws-public-ipv4-address-charge-public-ip-insights/

Reply:
Yes, please see this ticket for more details: https://github.com/kubernetes/k8s.io/issues/5759